examples/combined-odata-mock/app.js

```diff
@@ -14,6 +15,9 @@ var runAsync = async () => {
     try {
         app.use(await odataApp.init("./varkes_config.json"))
         app.use(await connectorApp.init("./varkes_config.json"))
+        app.use(await cockpitApp.init())
+        app.use(cors())

```

Comment:
why we need cors? If really needed, can it be configured in cockpit module?

Reply:
removed